src/Tizen.Network.Connection/Tizen.Network.Connection/CellularProfile.cs

```diff
@@ -253,6 +253,26 @@ public CellularPdnType RoamingPdnType
             }
         }
 
+		/// <summary>
+		/// Checks whether the connection is in roaming state.
+		/// </summary>
+		/// <since_tizen> 6 </since_tizen>
+		/// <value> True if the cellular profile is in roaming state, otherwise false.</value>
+		public bool IsRoaming
```

Comment:
Please use the same indentation as the rest of the file (spaces instead of tabs).